src/PayID/pay-id-admin-client.ts

```diff
@@ -0,0 +1,78 @@
+import PayIDError from './pay-id-error'
+
+/* eslint-disable @typescript-eslint/require-await */
+// TODO(keefertaylor): Enable lint rules when method is implemented.
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable class-methods-use-this */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable max-classes-per-file */
+
+/**
+ * An admin for Xpring's PayID service.
+ *
+ * Admin clients can create, update, and delete address mappings. To resolve PayIDs to addresses, please use `PayIDClient`.
+ *
+ * Inputs to this class are always in X-Addresses.
+ * @see https://xrpaddress.info/
+ *
+ * @warning This class is experimental and should not be used in production applications.
+ * TODO(keefertaylor): Export this class in index.ts when it's ready for external consumption.
+ */
+export default class PayIDAdminClient {
+  private static putEndpoint = '/v1/users'
```

Comment:
This feels weird to me.

You take a `remoteURL`, implying you support PayID services besides the Xpring PayID service, but you hardcode this endpoint, which may be different for other PayID services. Thoughts?

Reply:
I'm going to merge this PR and we can talk about this later. 

Semantically I'd assume a payID server would just work with this. Why would we want this to be configurable?